content/services/cloudcoach.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Cloud Coach"
+type: "services"
+subtitle: ""
+description: "We help you be successful in the Cloud by being involved advisors and sparring partners. We will also help you should things go wrong."
+icon: developer.svg
+---
+
+{{<section>}}
+{{% col-left-10 title="Cloud Coach" %}}
+There is great value in cloud technology for startups, but having a dedicated skilled Platform Team often isn’t financially viable until they become scale-ups. With our Cloud Coach service, we help your engineers leverage cloud technology effectively by being involved advisors and sparring partners. We help you make the right decisions, gather the right skills, and build the right things the right way; all while also offering active support should things go wrong.
+
+
+Our Cloud Coach service helps startups be successful in using cloud technology in a financially sustainable way. Our service consists of 2 parts;
+
+- Hands-off Coaching
```

Comment:
Opinion: Any list that only has two entries should probably not be presented as a list at all.

Suggestion: "Our service consists of both Hands-off Coaching to make sure you'll able to manage your platform yourself, as well as Slack Support when solving those daily challenges really need a second, third or even fourth pair of eyes to make sure the right decisions get made." or something along those lines.

Reply:
I think we can even leave it out altogether. Otherwise we mostly end up repeating what the paragraph above already said.